opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -239,6 +230,36 @@ namespace cpgrid
             // Make the grid.
             processEclipseFormat(g, nnc_cells, z_tolerance, false, turn_normals);
         }
+
+        {
+            auto format_ijk = [this](int cell) -> std::string {
+                std::array<int,3> ijk;
+                this->getIJK(cell, ijk);
+                return std::string{"("} + std::to_string(ijk[0] + 1) + std::string{","} + std::to_string(ijk[1] + 1) + std::string{","} + std::to_string(ijk[2] + 1) + std::string{")"};
+            };
+
+
+            auto make_warning = [this, &format_ijk](const std::string& reason, const Opm::KeywordLocation& location, const Opm::NNCdata& nnc) -> std::string {
+                return std::string{"Problem with NNC connection \n"} +
```

Comment:
As the `C` in `NNC` means "connection", this is slightly redundant (e.g., ATM machine).  We should also remove the trailing blank before the newline.